docs/extending.md

```diff
@@ -258,6 +269,34 @@ conflicts.
 [['my-option'], ...]
 ```
 
+## Dynamic Groups
+
+Caluma allows you to write code that returns groups for use in the `address_groups` and `control_groups` properties of a `Task`.
```

Comment:
The groups are used in `addressed_groups` / `controlling_groups`. The code is used in `address_groups` / `control_groups`.

Reply:
True, but the transforms can only be used in the task. Maybe a completely different wording would be better, do you have a proposal?

Reply:
```suggestion
Caluma allows you to write code to be executed as part of the JEXL evaluation in the `address_groups` and `control_groups` properties of a `Task`.
```